modules/samtools/bamtocram/main.nf

```diff
@@ -0,0 +1,35 @@
+//There is a -L option to only output alignments in interval, might be an option for exons/panel data?
+process SAMTOOLS_BAMTOCRAM {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::samtools=1.15.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/samtools:1.15.1--h1170115_0' :
+        'quay.io/biocontainers/samtools:1.15.1--h1170115_0' }"
+
+    input:
+    tuple val(meta), path(input), path(index)
+    path  fasta
+    path  fai
```

Comment:
Always wondered why these don't go together... but w/e

Reply:
Some tools needs just one, some the two, so more difficult to actually combine and give out, much easier to give out separately